src/components/Search/Search.scss

```diff
@@ -1,14 +1,14 @@
-@import "./../../css/common.scss";
+@import "./../../css/variables.scss";
 
 .support-search {
```

Comment:
I don't like this name, the component is called Search and the css class support-search. It feels a bit confusing. What about renaming it to be just search?

Reply:
Can it be search?

Reply:
The class was renamed